GagarinDmitry/src/Services/Identity/Identity.Api/Extensions/Options/LoggerConfiguration.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Serilog;
+using Serilog.Exceptions;
+using Serilog.Sinks.Elasticsearch;
+using System.Reflection;
+
+namespace Identity.WebApi.Extensions.Options
+{
+    /// <summary>
+    /// Provides LoggerConfiguration.
+    /// </summary>
```

Comment:
If you decided to use documentation, please keep tracking of adding it everywhere)